modules/dynamic-plugins/proc-rhdh-example-external-dynamic-plugins.adoc

```diff
@@ -37,8 +37,8 @@ global:
                       text: Notifications
                     config:
                       pollingIntervalMs: 5000   
-      - package: '@janus-idp/backstage-scaffolder-backend-module-kubernetes-dynamic@1.3.5'
-        # https://registry.npmjs.org/@janus-idp/backstage-scaffolder-backend-module-kubernetes-dynamic
+      - package: '@backstage-community/plugin-scaffolder-backend-module-kubernetes-dynamic@1.3.5'
```

Comment:
Should this be the latest 2.2.2 or some older version?

```suggestion
      - package: '@backstage-community/plugin-scaffolder-backend-module-kubernetes-dynamic@2.2.2'
```

Reply:
Seems like 2.2.2 is the correct version; however https://github.com/backstage/community-plugins/blob/main/workspaces/scaffolder-backend-module-kubernetes/backstage.json says Backstage 1.33, not 1.32 -- @BethGriggs @04kash Thoughts?

Reply:
Looks like while migrating the plugin there was actually nothing done to update it to 1.33. It still seems to be on 1.32.5 (https://github.com/backstage/community-plugins/commit/599be1ea0ade89eb3429caf6167469d260855438#diff-c7aedd3b99375405eeabbea7a17a818cce77164b592a56bf36b5fbd983fc6afe) . [This file](https://github.com/backstage/community-plugins/blob/main/workspaces/scaffolder-backend-module-kubernetes/backstage.json)  says 1.33 because the [create-workspace](https://github.com/backstage/community-plugins/blob/main/package.json#L10) command automatically updates backstage.json to the latest version without checking what versions the plugins under that workspace support. 

So yeah, 2.2.2 is the correct version

Reply:
Based on what @batzionb mentioned [here](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/824#discussion_r1895037592), since the migrated plugins don't have the @janus-idp/cli dependency anymore, idts it can be installed this way. 

There is a PR up to add a wrapper to showcase for this plugin: https://github.com/janus-idp/backstage-showcase/pull/2085 which I think will be included for 1.5. So for 1.5 we can remove this plugin from `modules/dynamic-plugins/proc-rhdh-example-external-dynamic-plugins.adoc` and would be included here: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.4/html/dynamic_plugins_reference/index

For 1.4, I think it would be good to check with the dynamic plugins team and see what they think.

Reply:
Do we want to have here `plugin-notifications-backend` if it was added as a wrapper for 1.4?

Reply:
if it's included in the list of BUNDLED dynamic plugins then no, it doesn't need to also be listed as an EXTERNAL one.